lib/stat_tracker.rb

```diff
@@ -108,4 +108,26 @@ def lowest_total_score
     end
     scores.min
   end
+  def coach_win_percentages
+    coach_games = Hash.new { |hash, key| hash[key] = { wins: 0, games: 0}}
+    @all_game_teams.each do |game_team|
+      coach = game_team.head_coach
+      coach_games[coach][:games] += 1
+      coach_games[coach][:wins] += 1 if game_team.result == "WIN"
+    end
+      coach_games.transform_values do |stats| 
+        games = stats[:games]
+        games > 0 ? ((stats[:wins].to_f / games) * 100).round : 0
+      end
+  end
+
+  def winningest_coach
+    coach_win_percentages = coach_win_percentages
```

Comment:
What is the purpose of this line?

Reply:
it looks like an error because it is trying to assign to a method.

Reply:
In Ruby, if you want to call a method that has no arguments, you need to include parentheses when invoking it, especially if you are assigning the return value to a variable with the same name as the method.

Reply:
Are you sure @Montana-Pfeifer ?
<img width="567" alt="image" src="https://github.com/user-attachments/assets/1a9e2049-b448-487b-bbe6-cd6999ff70c6">